vector/src/main/java/im/vector/app/features/call/VectorCallViewEvents.kt

```diff
@@ -30,6 +30,8 @@ sealed class VectorCallViewEvents : VectorViewEvents {
     object ShowDialPad : VectorCallViewEvents()
     object ShowCallTransferScreen : VectorCallViewEvents()
     object FailToTransfer : VectorCallViewEvents()
+    object ShowScreenSharingPermissionDialog : VectorCallViewEvents()
+    object StopScreenSharingService : VectorCallViewEvents()
```

Comment:
Optional: This is a good opportunity to get rid of the commented code below. They're nearly 2 years old, I don't think we'll need it

Reply:
Right :) done.